src/main/java/org/aerogear/digger/client/DiggerClient.java

```diff
@@ -147,11 +147,15 @@ public DiggerClient build() throws DiggerClientException {
      * @param name      job name that can be used later to reference job
      * @param gitRepo   git repository url (full git repository url. e.g git@github.com:wtrocki/helloworld-android-gradle.git
      * @param gitBranch git repository branch (default branch used to checkout source code)
+     * @param storeBuildsDays the number of days a build should be persisted for before cleanup
+     * @param storeBuildsTotal the total number of builds that should be stored for this job
+     * @param storeArtifactsDays the number of days each build artifact should be persisted for before cleanup
+     * @param storeArtifactsTotal the total number of build artifact that should be stored for this job
      * @throws DiggerClientException if something goes wrong
      */
-    public void createJob(String name, String gitRepo, String gitBranch) throws DiggerClientException {
+    public void createJob(String name, String gitRepo, String gitBranch, int storeBuildsDays, int storeBuildsTotal, int storeArtifactsDays, int storeArtifactsTotal) throws DiggerClientException {
```

Comment:
those extra params makes this method to have too many params. I think it will  make it cleaner if we group those extra params into a single object?